s3/Module.md

```diff
@@ -0,0 +1,59 @@
+## Overview
+The module provides the capability to manage buckets and objects in [AWS S3](https://aws.amazon.com/s3/).
+
+This module supports [Amazon S3 REST API](https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html) `2006-03-01` version.
+ 
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [AWS account](https://portal.aws.amazon.com/billing/signup?nc2=h_ct&src=default&redirect_url=https%3A%2F%2Faws.amazon.com%2Fregistration-confirmation#/start)
+- Obtain tokens - Follow [this link](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+
+## Quickstart
+To use the AWS S3 connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+First, import the `ballerinax/aws.s3` module into the Ballerina project.
+```ballerina
+import ballerinax/aws.s3;
+```
+
+### Step 2: Create a new connector instance
+Create client using connection configuration.
```

Comment:
Is this sentence needed? Doesn't the next sentence cover what this says?

Reply:
No. Not needed. Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/pull/55/commits/df2a25cd27ea5b50683e8473e083cb5f4a90927b